src/css/profile/mobile/themes/dark.variables.less

```diff
@@ -43,3 +43,4 @@
 
     @divider-color: #d4d4d4;
     @divider-opacity: 15%;
+    @subheader-divider-color: #fafafa;
```

Comment:
this is the same as: @color-whie, so
`@subheader-divider-color: @color-white`

Reply:
I moved the @color-white to the "base.less". and change it use the @color-white.